comp/metadata/host/hostimpl/utils/meta.go

```diff
@@ -37,6 +37,7 @@ type Meta struct {
 	ClusterName               string   `json:"cluster-name,omitempty"`
 	LegacyResolutionHostname  string   `json:"legacy-resolution-hostname,omitempty"`
 	HostnameResolutionVersion int      `json:"hostname-resolution-version,omitempty"`
+	CCRID                     string   `json:"ccrid,omitempty"`
```

Comment:
Given that the other fields on this struct are rather long, what do you think about replacing the field name with a non-acronym version: `CanonicalCloudResourceID`? We can still use `ccrid` for the serialization format. It should help with readability for future devs that are not as familiar with the acronym.